odoo/addons/base/models/ir_model.py

```diff
@@ -1969,7 +1969,10 @@ def _compute_complete_name(self):
     @api.depends('model', 'res_id')
     def _compute_reference(self):
         for res in self:
-            res.reference = "%s,%s" % (res.model, res.res_id)
+            reference = ''
+            if res.model and res.id != 0:
+                reference = "%s,%s" % (res.model, res.res_id)
```

Comment:
You can replace it, with a f-string plz ?

Reply:
@ryv-odoo updated, if you dont mind, can you help to write the test case

Reply:
up @ryv-odoo, he needs some help for writing a test